tools/sweer/bin/click_mouse

```diff
@@ -0,0 +1,41 @@
+#!/root/python3.11/bin/python3
```

Comment:
the shebangs are currently mixed. Some say #!/usr/bin/env python3, others do the /usr/bin/env